seeder/src/seeder_flet/lib/views/login.py

```diff
@@ -1,12 +1,15 @@
+import asyncio
 import flet as ft
 
 __all__ = ["login_view"]
 
 
-def handle_login_click(e, username: str, password: str, backend: str):
+async def handle_login_click(username: str, password: str, backend: str):
     "Try to login with backend"
     ...
 
+def handle_login_click_wrapper(e, username: str, password: str, backend: str):
+    asyncio.run(handle_login_click(username, password, backend))
```

Comment:
Why is it like this?

Just use 

```python
await handle_login_click(username, password, backend)
```

Reply:
Resolved on https://github.com/coreproject-moe/CoreProject/pull/3863/commits/f5d35e04aa1dc2cdf25708ddf70f48278c9c58b6